res/database/Default/user/scr0llbaer.xml

```diff
@@ -1764,6 +1784,152 @@ So that's 4120 per episode.</en>
 			<speed min="30" max="80" />
 			<studio_size value="2" />
 		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="71f4752c-2c8f-41f0-82c6-3461e02e8160" creator="9551" comment="TODO: To test a future 'franchise script' feature. Episode titles and contents by ChatGPT. A bit formulaic perhaps.">
+			<title>
+				<de>%ROLENAME1% und %ROLENAME2% - Die %new% %adventures%</de>
+				<en>%ROLENAME1% and %ROLENAME2% - The %new% %adventures%</en>
+			</title>
+			<description>
+				<de>Eine Neuauflage der beliebten tschechischen Marionettenserie über Vater %ROLENAME1%, den kleinen Sohn %ROLENAME2%, Schulfreundin %ROLENAME4%, Großmutter Frau %ROLENAME3% und Hund %ROLENAME5%. Behutsam modernisiert und mit nur sehr spärlicher Anwendung neuer technischer Hilfsmittel, so dass der Charme erhalten bleibt.</de>
+				<en>A new edition of the popular Czech puppet series about father %ROLENAME1%, little son %ROLENAME2%, school girl friend %ROLENAME4%, grandmother Ms. %ROLENAME3% and dog %ROLENAME5%. Cautiously modernized and with only very sparse use of new technical aids, so that the charm is preserved.</en>
+			</description>
+
+			<variables>
+				<new>
+					<de>Neuen|Neuartigen|Weiteren|Frischen|Ungeahnten|Nachfolgenden|Unversuchten|Ursprünglichen|Neuerlichen|Wiederbelebten|Erneuten|Neugeborenen|Wiedererstarkten|Reanimierten|Auferstandenen</de>
+					<en>New|Novel|Further|Fresh|Unprecedented|Successive|Untried|Pristine|Refreshed|Revived|Renewed|Reborn|Invigorated|Reanimated|Resurrected</en>
+				</new>
+				<adventures>
+					<de>Abenteuer|Erlebnisse|Zeiten|Tücken|Ereignisse|Entdeckungen|Eskapaden|Erforschungen|Missionen|Vorhaben|Episoden|Taten|Prüfungen|Aufgaben|Unternehmungen|Schwierigkeiten|Gefahren|Herausforderungen</de>
+					<en>Adventures|Experiences|Times|Ordeals|Happenings|Explorations|Escapades|Expeditions|Missions|Ventures|Episodes|Deeds|Trials|Quests|Acts|Enterprises|Tribulations|Hazards|Challenges</en>
```

Comment:
English and German have different count of options: no German version for "Acts"?

Reply:
done